packages/server/src/extensions/devicePublicKey/isRecognizedDevice.ts

```diff
@@ -90,9 +90,6 @@ export async function isRecognizedDevice(
           throw new Error('DevicePublicKey attestation could not be verified.');
```

Comment:
This `Error` should reveal why the equality check failed:
```suggestion
          const _err = err as Error;
          throw new Error(`DevicePublicKey attestation could not be verified: ${_err.message}`);
```